modules/afsmtp/afsmtp.c

```diff
@@ -45,6 +45,7 @@ typedef struct
 {
   gchar *phrase;
   gchar *address;
+  LogTemplate *addr_tmpl;
   afsmtp_rcpt_type_t type;
```

Comment:
Well, all of Subject(), Body() and Header() are following this pattern. All of them contains its own string (from the parsed syslogconf) and its LogTemplate struct for the compiled string.
Of course the parsing functions could compile the strings from the syslogconf directly into the LogTemplate struct and that would eventuate that the recipient structs wouldn't need the gchar *address, but as I mentioned that would break the consistency between the smtp templating methods.